### DISCUSSION TITLE
Add 'Unified Profiles' icon to paste library

### DISCUSSION THREAD

**Discussion by apombo-ux** [twilio-labs/paste#3934] [Request: Icon] [answered]
### Description

The product marketing team has designed an icon for the Twilio Unified Profiles product. We can use the SVG in the project directly for now, but I wanted to start the process of getting it incorporated into paste resources. This ticket is to see what else I need to do to make that happen.

Following the Contributing an Icon guidelines for [setting up an icon in figma](https://paste.twilio.design/introduction/contributing/icons#setting-up-an-icon-in-figma) and [formatting an icon](https://paste.twilio.design/introduction/contributing/icons#formatting-an-icon), I've added the icon to the Paste Components & Themes library on a [Unified Profiles icon branch](https://www.figma.com/design/wOgYPbS734SAm0FOco0cqo/Unified-Profiles-icon?node-id=40498-141&t=3XBnNsHqV5oHvaEG-4). 

I've confirmed that it exports as a clean SVG, and I'm planning to work with my development team to follow the [self-service](https://paste.twilio.design/introduction/contributing/icons#self-service-instructions) path for adding the icon to the paste repo, but it hasn't been added yet.

Besides having someone on the paste team check and merge my branch in if it looks alright (unless that needs to wait until the icon is in the code repository?), is there anything else I should do?

Thanks!

### Link to icon
Product marketing source:
https://library.twilio.com/media/?mediaId=5F9824E1-588F-42EA-9665BBA75525C547

Added to Figma:
https://www.figma.com/design/wOgYPbS734SAm0FOco0cqo/Unified-Profiles-icon?node-id=40498-141&t=3XBnNsHqV5oHvaEG-4

SVG export:
![ProductUnifiedProfiles](https://github.com/twilio-labs/paste/assets/98772241/cbcb360c-c1c9-4706-9466-5047c0e991a4)

### Icon type

- [ ] UI icon
- [X] Product icon

**Comment by serifluous** [comment, marked as answer]
Hey @apombo-ux ! Just reviewed the Figma branch and merged it in. The icon's on track to be added in code by the end of the month, but let me know if your engineering team would rather go the self-service route and get it added in faster.

**Reply by serifluous** [reply]
Hey @apombo-ux  ! The icon is now available in Paste. Check it out here: https://paste.twilio.design/components/icon
